packages/flutter/lib/src/rendering/animated_size.dart

```diff
@@ -99,8 +99,18 @@ class RenderAnimatedSize extends RenderAligningShiftedBox {
     );
   }
 
+  /// The animation controller that is used to drive the resizing.
+  ///
+  /// The animation itself is exposed by the [animation] property.
+  @visibleForTesting
+  AnimationController get controller => _controller;
```

Comment:
This is a public API not only for testing, right?

We can refer to the implementation of `DragGestureRecognizer.debugLastPendingEventTimestamp` and document that it is only available for testing.

Reply:
Previously `_controller` and `_animation` were private because `AnimatedSize` is a kind of implicit animation, so users don't need to have access to the animation controller. 
Are you sure that we need to expose them to the users and create additional getters for testing only?

Reply:
I got your point, thanks. 
Could you please review the latest commit?

Reply:
There is a problem with the CI infra, let's wait for it to be fixed.